File: Listas antigas em C/Lista 1/Questão 9 Lista 1.c
/*
Questão 9 Lista 1
Faça um algoritmo para ler um número inteiro, depois calcular e imprimir a sua raiz quadrada e sua potenciação.
*/
#include <stdio.h>
#include <math.h>

int main(void){
	int numero;
	float raiz, potencia;
  		
	printf("digite o numero\n");
	scanf("%f", &numero);
	raiz = sqr(numero);
	printf("raiz quadrada = %f", raiz);
	potencia = pow(numero, 2);
	printf("Quadrado = %f", potencia);
	return 0;
}

File: Lista 1/Questão 9 Lista 1.py
"""
Questão 9 Lista 1
Faça um algoritmo para ler um número inteiro, depois calcular e imprimir a sua raiz quadrada e sua potenciação.
"""
from math import sqrt
numero=int(input("Digite o numero: "))
raiz = sqrt(numero)
print(f"raiz quadrada = {raiz}")
potencia = numero**2
print(f"Quadrado = {potencia}")
